File: myblog-server/src/main/java/com/site/blog/service/ImgService.java
package com.site.blog.service;

import com.baomidou.mybatisplus.extension.service.IService;
import com.site.blog.model.entity.Img;
import org.springframework.web.multipart.MultipartFile;

/**
 * @author yanni
 * @date time 2021/10/29 12:46
 * @modified By:
 */
public interface ImgService extends IService<Img> {

     Img uploadImage(MultipartFile file);
     void deleteImage(String fileName);
     void createDirectory(String path);
}

File: myblog-jpa/src/main/kotlin/com/site/blog/service/ImgService.kt
package com.site.blog.service


import com.site.blog.model.entity.Img
import org.springframework.web.multipart.MultipartFile

/**
 * @author yanni
 * @date time 2021/10/29 12:46
 * @modified By:
 */
interface ImgService  {
    fun uploadImage(file: MultipartFile): Img
    fun deleteImage(fileName: String)
    fun createDirectory(path: String)
   fun getById(id: Long): Img
    fun removeById(id: Long)
}